Mathlib/Analysis/NormedSpace/OperatorNorm.lean

```diff
@@ -394,11 +394,6 @@ theorem op_norm_comp_le (f : E →SL[σ₁₂] F) : ‖h.comp f‖ ≤ ‖h‖ *
       exact h.le_op_norm_of_le (f.le_op_norm x)⟩
 #align continuous_linear_map.op_norm_comp_le ContinuousLinearMap.op_norm_comp_le
 
--- Porting note: restatement of `op_norm_comp_le` for linear maps.
-/-- The operator norm is submultiplicative. -/
-theorem op_norm_comp_le' (h : Eₗ →L[𝕜] Fₗ) (f : E →L[𝕜] Eₗ) : ‖h.comp f‖ ≤ ‖h‖ * ‖f‖ :=
```

Comment:
what is the reason to delete this?

Reply:
It's syntactically a special case of the above

Reply:
I imagine this was added only to aid the port when Lean wouldn’t automatically infer the correct type. I don’t see a use for this now that things “just work.”

Reply:
Yeah I was just confused as it wasn't used anywhere it seems? So why would the porter add it, oh well if it seems useless its good to get rid of it